src/python/pants/task/testrunner_task_mixin.py

```diff
@@ -63,6 +66,106 @@ def execute(self):
       all_targets = self._get_targets()
       self._execute(all_targets)
 
+  def report_test_info(self, scope, target, key, test_info):
+    """
+    Add test information to target information
+    :param string scope: The running scope
+    :param Target target: The target that we want to store the test info under
+    :param list of strings or string key: The key for the info being stored
```

Comment:
In the context of tests, I think explicitly changing the `key` parameter to a string `test_name` is fine... the key-list bit is primarily important for generality in the RunTracker API, but less so here.